magnum-zio/src/main/scala/com/augustnagro/magnum/magzio/Transactor.scala

```diff
@@ -1,36 +1,32 @@
 package com.augustnagro.magnum.magzio
 
-import com.augustnagro.magnum.{DbCon, DbTx, SqlException, SqlLogger}
+import com.augustnagro.magnum.{
+  DbCon,
+  DbTx,
+  SqlException,
+  SqlLogger,
+  TransactorOps
+}
 import zio.{Semaphore, Task, Trace, UIO, ULayer, ZIO, ZLayer}
 
 import java.sql.Connection
 import javax.sql.DataSource
 import scala.util.control.NonFatal
 
-class Transactor private (
-    dataSource: DataSource,
-    sqlLogger: SqlLogger,
-    connectionConfig: Connection => Unit,
-    semaphore: Option[Semaphore]
-):
+final case class Transactor(
+    private val dataSource: DataSource,
+    private val sqlLogger: SqlLogger,
+    private val connectionConfig: Connection => Unit,
+    private val semaphore: Option[Semaphore]
```

Comment:
Indeed these should be `private` 🙂

Reply:
They are already private:

https://docs.scala-lang.org/tour/classes.html#:~:text=Parameters%20without%20val%20or%20var%20are%20private%20values%2C%20visible%20only%20within%20the%20class.